editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -99,8 +99,7 @@ void SpatialEditorViewport::_update_camera(float p_interp_delta) {
 			// We interpolate a different point here, because in freelook mode the focus point (cursor.pos) orbits around eye_pos
 			camera_cursor.eye_pos = old_camera_cursor.eye_pos.linear_interpolate(cursor.eye_pos, CLAMP(factor, 0, 1));
 
-			float orbit_inertia = EDITOR_GET("editors/3d/navigation_feel/orbit_inertia");
-			orbit_inertia = MAX(0.0001, orbit_inertia);
+			const float orbit_inertia = MAX(0.0001, (float)EDITOR_GET("editors/3d/navigation_feel/orbit_inertia"));
```

Comment:
Shouldn't this min value of `0.0001` be handled through the property hints instead of hardcoded here and below?